docs/community/model-providers/mlx.md

```diff
@@ -0,0 +1,238 @@
+# MLX (Apple Silicon)
+
+{{ community_contribution_banner }}
+
+!!! info "Language Support"
+    This provider is only supported in Python.
+
+!!! warning "Platform Requirement"
+    MLX is designed specifically for Apple Silicon (M1/M2/M3/M4 chips) and CUDA environments. Requires Python ≤3.13 on macOS or Linux.
+
+[MLX](https://github.com/ml-explore/mlx) is Apple's machine learning framework optimized for Apple Silicon. The `strands-mlx` provider enables local inference, LoRA fine-tuning, and vision capabilities on Apple Silicon devices with support for mlx-community quantized models.
+
+## Installation
+
+To use MLX models with Strands Agents on Apple Silicon:
+
+```bash
+# Create virtual environment with Python 3.13
+uv venv --python 3.13 && source .venv/bin/activate
+
+# Install dependencies
+uv pip install strands-mlx strands-agents-tools
+```
+
+Or with pip:
+
+```bash
+pip install strands-mlx strands-agents-tools
+```
+
+## Quick Start
+
+### Basic Text Inference
+
+```python
+from strands import Agent
+from strands_mlx import MLXModel
+from strands_tools import calculator
+
+model = MLXModel(model_id="mlx-community/Qwen3-1.7B-4bit")
+agent = Agent(model=model, tools=[calculator])
+
+agent("What is 29 * 42?")
+```
+
+### Vision Models
```

Comment:
An intro text blurb might be useful in these two sections.  E.g. "MLX also supports vision models which can understand/parse images, videos, and audio"

Should this agent also have a `read_file` so that it can grab the given files?

Reply:
I'm confused in this example then - when I run this:

```python
agent("Describe: <image>photo.jpg</image>")
```

where is the photo coming from?

Reply:
It's the path of file in xml tags